### PR TITLE
Add API to retrieve Load Balancer Health information

### PR DESCRIPTION
**Pull request by anbrenna**

### CODE REVIEW

File: model/oci_recommendation.go [hunk 1]
```diff
@@ -25,3 +25,25 @@ type Recommendation struct {
...
+const (
+	RecommendationTypeBlockStorage        = "Block Storage"
+	RecommendationTypeComputeInstanceIdle = "Compute Instance Idle"
+	RecommendationTypeOldSnapshot         = "Old Snapshot"
+	RecommendationTypeUnusedResource      = "Unused Resource"
+	RecommendationTypeUnusedStorage       = "Unused Storage"
+)
+
+/*
+	Compute Instance Under-utilized
```
Comment (cirelli94) [inline model/oci_recommendation.go:45]: What is this comment? Do we need it?

File: thunder-service/service/oci_compartments.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			continue
+		}
+
+		var compTmp model.OciCompartment
+		for _, s := range resp.Items {
+			compTmp.CompartmentId = *s.Id
+			compTmp.Name = *s.Name
+			compTmp.Description = *s.Description
+			compTmp.TimeCreating = s.TimeCreated.String()
+			// fmt.Println(i, *s.Id, *s.Name, *s.Description, *s.TimeCreated)
```
Comment (cirelli94) [inline thunder-service/service/oci_compartments.go:82]: Please, remove it :)

**Review by cirelli94 — CHANGES_REQUESTED** [review]

File: thunder-service/service/oci_compartments.go (hunk 2 above)
Comment (anbrenna) [reply thunder-service/service/oci_compartments.go:82]: Comment removed

File: model/oci_recommendation.go (hunk 1 above)
Comment (anbrenna) [reply model/oci_recommendation.go:45]: It was a typo. I removed it

**Pull request merged**